editor/plugins/shader_editor_plugin.h

```diff
@@ -33,15 +33,20 @@
 
 #include "editor/code_editor.h"
 #include "editor/editor_plugin.h"
+#include "editor/plugins/visual_shader_editor_plugin.h"
+#include "editor/shader_create_dialog.h"
 #include "scene/gui/menu_button.h"
 #include "scene/gui/panel_container.h"
 #include "scene/gui/rich_text_label.h"
+#include "scene/gui/split_container.h"
```

Comment:
All these new includes are not necessary. You can forward-declare and move them to cpp file.

Reply:
Makes sense!